sdcm/tester.py

```diff
@@ -300,7 +300,15 @@ def __init__(self, *args, **kwargs):  # pylint: disable=too-many-statements,too-
         self.log.debug("IP used for SSH connections is '%s'",
                        ip_ssh_connections)
         self.test_config.set_ip_ssh_connections(ip_ssh_connections)
-        self._duration = self.params.get(key='test_duration')
+
+        self._stress_duration: int = self.params.get('stress_duration')
+        self._prepare_duration: int = self.params.get('prepare_duration')
+        if self._stress_duration > 0:
```

Comment:
you don't need this check here, because if this is the default (0) it won't add anything, but if it has any other value, you will sum it up...
and if someone creates this parameter with negative, this is operator error, but perhaps could be enforced in the `sct_config` setting for this parameter

Reply:
or perhaps it should be `if not self._duration`

Reply:
Changed